example/demo.md

```diff
@@ -0,0 +1,200 @@
+# Kai Demo
+
+<!-- markdownlint-disable-file MD014 -->
+
+Konveyor AI (kai) is Konveyor's approach to easing modernization of application
+source code to a new target by leveraging LLMs with guidance from static code
+analysis augmented with data in Konveyor that helps to learn how an Organization
+solved a similar problem in the past.
+
+- [Kai Demo](#kai-demo)
+  - [Overview](#overview)
+  - [Prerequisites](#prerequisites)
+  - [Step 1: Setup](#step-1-setup)
+    - [Running Kai with IBM-served Llama 3](#running-kai-with-ibm-served-llama-3)
+    - [Running Kai with Amazon Bedrock-served Llama 3](#running-kai-with-amazon-bedrock-served-llama-3)
+    - [Running Kai with GPT-3.5-Turbo](#running-kai-with-gpt-35-turbo)
+    - [Running Kai with Cached Responses Only (demo mode)](#running-kai-with-cached-responses-only-demo-mode)
+  - [Step 2: Clone the coolstore app](#step-2-clone-the-coolstore-app)
+  - [Step 3: Run Kai](#step-3-run-automated-usage-of-kai)
+
+## Overview
+
+In this demo, we will showcase the capabilities of Konveyor AI (Kai) in
+facilitating the modernization of application source code to a new target. We
+will illustrate how Kai can handle various levels of migration complexity,
+ranging from simple import swaps to more involved changes such as modifying
+scope from CDI bean requirements. Additionally, we will look into migration
+scenarios that involves EJB Remote and Message Driven Bean(MBD) changes
+
+We will focus on migrating a partially migrated [JavaEE Coolstore
+application](https://github.com/konveyor-ecosystem/coolstore.git) to Quarkus, a
+task that involves not only technical translation but also considerations for
+deployment to Kubernetes. By the end of this demo, you will understand how
+Konveyor AI (Kai) can assist and expedite the modernization process.
+
+This demo will focus primarly in driving Kai's RPC server via helper scripts.
+This guide will not cover the IDE integration.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Git](https://git-scm.com/downloads)
+- [GenAI credentials](../docs/llm_selection.md#ibm-bam-service)
+- [Maven](https://maven.apache.org/install.html)
+- Quarkus 3.10
+- Java 17
+
+## Step 1: Setup
+
+[You can configure Kai in multiple ways](../docs/contrib/configuration.md). The best
```

Comment:
This still references config.toml instead of the Konveyor extension settings in the UI or the settings.json file.